### ISSUE TITLE
Responses are not intercepted on page reload

### ISSUE THREAD

**Issue by churixho** [code-mancers/interceptor#161]
My SPA site is built using ReactJS + react router. 
Let's say we have page A whichs load a.json and page B which loads b.json

Case 1: Go to page A; intercept a.json;  then refresh the page
Results: a.json is not intercepted

Case 2: Go to page A; intercept b.json; then go to page B
Results: b.json is intercepted successfully

Expected behaviour: Response could be intercepted upon page refresh

**Comment by revathskumar** [comment]
@churixho Chrome/Firefox?

**Comment by churixho** [comment]
Chrome

**Comment by churixho** [comment]
It appears to me that the this is due to race condition. When I delay the axios in my site (i.e. the loading of a.json in the example) for 2 seconds, the result could be interruped.

**Comment by amit-mb** [comment]
@churixho Yes, the issue here is that the the web page gets the data from `axios` before Interceptor can inject a script. We are working towards a fix for this.

**Comment by therisingindian** [comment]
I'm also waiting for the capability to intercept response on page reload.

**Comment by quinncomendant** [comment]
I'm using Interceptor to debug an angular page, and need to intercept XHR requests run during page load. I too notice that Interceptor can't intercept this when reload the page.
